### ISSUE TITLE
Subtitle handling issue

### ISSUE THREAD

**Issue by prilly** [steeve/plugin.video.pulsar#91]
Subtitles is stored in /temp eventhough the main setting in kodi is set to "Store subtiles with the movie" this is probably becouse torrent2http is caching something here, or kodi sees the file as stored here? the problem with this is when you use a stoarge path for download like /storage/movie the movie folder itself is stored here but the subtitle is not. evetualy the /temp folder will fill up with old subtitle .srt files requireing a manual removal. this is not good.

**Comment by HazCod** [comment]
+1 on this

**Comment by hazcod** [comment]
@prilly is this still an issue?

**Comment by ghost** [comment]
yes this is still a issue. It have been nice to see the subtitles beeing stored in the same folder as the downloading media. if pulsar is setup to delete the media after download it will also delete the added subtitles. the problem now is as indicated abow, the temp folder is increasing with subtitle files for media that has been delete.

**Comment by hazcod** [comment]
@steeve any plans to fix this?

**Comment by nunofgs** [comment]
Also bit by this. Subtitles is not working on Android due to this.

Refs #150
